docs/data/material/customization/breakpoints/breakpoints.md

```diff
@@ -101,12 +49,24 @@ const theme = createTheme({
       tablet: 640,
       laptop: 1024,
       desktop: 1200,
+      // note that these values are defined in pixels
     },
   },
 });
 ```
 
-If you are using TypeScript, you would also need to use [module augmentation](/material-ui/guides/typescript/#customization-of-theme) for the theme to accept the above values.
+<!-- Keep in sync with packages/mui-system/src/createTheme/createBreakpoints.d.ts -->
+
+<!-- - [`theme.breakpoints.values`](/material-ui/customization/default-theme/?expand-path=$.breakpoints.values): Default to the [above values](#default-breakpoints). The keys are your screen names, and the values are the min-width where that breakpoint should start.
+- `theme.breakpoints.unit`: Default to `'px'`. The unit used for the breakpoint's values.
+- `theme.breakpoints.step`: Default to `5`. The increment divided by 100 used to implement exclusive breakpoints.
+  For example, `{ step: 5 }` means that `down(500)` will result in `'(max-width: 499.95px)'`.
```

Comment:
The step is used in all except the `up` one. It's directly used in `down` and `between`, and in `only` and `not` as they rely on `down` and `between` for their implementation.

I think we could keep it. It would be useful for advanced use cases.